library/pk_wrap.c

```diff
@@ -1099,9 +1097,37 @@ static int eckey_check_pair(const void *pub, const void *prv,
                             int (*f_rng)(void *, unsigned char *, size_t),
                             void *p_rng)
 {
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
Note: this can be simplified #7387 is merged.

Reply:
Ok, so this means that this PR has a dependency on a previous one. I'll set the proper label for this and add it in the main description ;)